builtin/providers/aws/resource_aws_elastic_beanstalk_environment.go

```diff
@@ -56,6 +56,10 @@ func resourceAwsElasticBeanstalkEnvironment() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"version_label": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
When you apply your example above, does this field show up in future diffs? It looks like it would need to be `Computed: true` as well